### ISSUE TITLE
Server crashes when 2 people join

### ISSUE THREAD

**Issue by epullay** [50DKP/FF2-Official#341]
Whenever I load a vsh map, and 2 or more people join, my server crashes. But it only happens with freak fortress enabled.

**Comment by 50Wliu** [comment]
Please make sure the following are up to date:
1. Metamod
2. Sourcemod
3. TF2Items
4. TF2Attributes
5. Freak Fortress 2

If you are confident that they are, please report the versions that you're using and I'll investigate further.

**Comment by epullay** [comment]
Yeah. They all are up to date. Just checked

**Comment by 50Wliu** [comment]
> please report the versions that you're using

I can't do anything without those.

**Comment by epullay** [comment]
Metamod: Source version 1.10.6
SourceMod Version: 1.8.0.5803
TF2Items: 1.6.4-279
TF2Attributes: 1.2.1
Freak Fortress: 1.10.8

**Comment by 50Wliu** [comment]
Thanks!

**Comment by Deathreus** [comment]
Try SM 1.7 instead, 1.8 still seems to have massive bugs

**Comment by 50Wliu** [comment]
@epullay can you make sure the TF2Attributes gamedata is also up-to-date?  Is there any information in the server console before the server crashes?

**Comment by epullay** [comment]
Nope. The server acts like if I restarted it. It displays no error messages
whatsoever. And TF2Attributes is up to date. It doesn't generate an error
message or anything. It simply restarts, it goes back to the map I have
specified on my server.cfg. No error message. I managed to run the server
without crashing it, but all models look like an error. But the server
downloads the models.

2015-12-30 23:12 GMT-03:00 Wliu <notifications@github.com>:

> @epullay <https://github.com/epullay> can you make sure the TF2Attributes
> gamedata is also up-to-date? Is there any information in the server console
> before the server crashes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/50DKP/FF2-Official/issues/341#issuecomment-168110760>.
>



-- 
*Saludos/Regards*

Ignacio Raglianti

Compositor, DJ, Actor de Voz y fundador de la comunidad TF2llay

Composer, DJ, Voice Actor and founder of the TF2llay community
